python_modules/dagster/dagster/_core/definitions/declarative_scheduling/scheduling_condition.py

```diff
@@ -228,16 +207,12 @@ def eager_with_rate_limit(
         any_parent_in_progress = SchedulingCondition.any_deps_match(
             SchedulingCondition.in_progress()
         )
-        failed_recently = SchedulingCondition.failed() & SchedulingCondition.scheduled_since(
```

Comment:
note that this is removed because I removed the ScheduledSinceCondition in this PR (as it got replaced by the more general TriggerSinceTargetCondition)